trafficserver_exporter/collector.py

```diff
@@ -172,6 +172,28 @@ def parse_metrics(self, data):
         )
         yield metric
 
+        # Incoming responses
+        metric = Metric(
+            "trafficserver_responses_incoming", "Incoming responses.", "counter"
```

Comment:
Should add `_total` onto these since they're ever-increasing counters.
`trafficserver_responses_incoming_total`

Reply:
Good point. Done!